### PR TITLE
pfs-23: HTMX spike integrated into latest

### PR DESCRIPTION
**Pull request by denialanderror**

### CODE REVIEW

File: web/template/layout/tab.gotmpl [hunk 1]
```diff
@@ -1,25 +1,11 @@
 {{ define "tab" }}
-  {{ template "page" . }}
+  <li class="moj-sub-navigation__item">
+    <div
+            class="moj-sub-navigation__link"
+            hx-get="{{ prefix . }}"
+            hx-target="#tab-content"
+            hx-push-url="/{{ . }}"
+    >{{ title . }}</div
+    >
```
Comment (Nicholassully) [inline web/template/layout/tab.gotmpl:9]: This could be on the line above or bring down the div?

File: internal/server/server.go [hunk 2]
```diff
@@ -12,19 +13,28 @@ import (
...
 type Template interface {
 	Execute(wr io.Writer, data any) error
+	ExecuteTemplate(wr io.Writer, name string, data any) error
 }
 
 func New(logger *zap.SugaredLogger, client ApiClient, templates map[string]*template.Template, envVars EnvironmentVars) http.Handler {
+	// middleware
+	// - logging
+	// - errors
+	// - vars
```
Comment (Nicholassully) [inline internal/server/server.go:29]: Do these still need to be here?

File: internal/server/app_vars_test.go [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package server
+
+import (
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+)
+
+func TestNewFinanceVars(t *testing.T) {
```
Comment (Nicholassully) [inline internal/server/app_vars_test.go:9]: Should this be changed to TestNewAppVars?

File: web/template/layout/tab.gotmpl (hunk 1 above)
Comment (denialanderror) [reply web/template/layout/tab.gotmpl:9]: I expect it is automated formatting. I'll see if I can adjust it.

**Review by Nicholassully — APPROVED** [review]

**Pull request merged**